# Game Parameters for Treasure Hunt
# These can be modified to tune game behavior

treasure_hunt_game:
  ros__parameters:
    # Number of rounds to play
    num_rounds: 10
    
    # Probability that treasure exists at an island (0.0 - 1.0)
    treasure_probability: 0.5
    
    # Distance to target in meters (6 feet = 1.83 meters)
    target_distance: 1.83

navigation_controller:
  ros__parameters:
    # Target distance in meters
    target_distance: 1.83

treasure_generator:
  ros__parameters:
    # Base treasure probability (can be randomized)
    treasure_probability: 0.5
    
    # Number of treasure hunt trials
    num_trials: 10
